doc/explanation/networks.md

```diff
@@ -1,7 +1,107 @@
+
+## Network devices
+
+Every instance must be assigned at least one network device, also called {abbr}`NIC (Network Interface Controller)`.
+You can configure the network device in one of the following ways:
+
+- Use the default network bridge that you set up during the LXD initialization.
+  Check the default profile to see the default configuration:
+
+        lxc profile show default
+
+  This method is used if you do not specify a network device for your instance.
+- Use an existing network interface by adding it as a network device to your instance.
+  This network interface is outside of LXD control.
+  Therefore, you must specify all information that LXD needs to use the network interface.
+
+  Use a command similar to the following:
+
+        lxc config device add <instance_name> <device_name> nic nictype=<nic_type> ...
+
+  See {ref}`instance_device_type_nic` for a list of available NIC types and their configuration properties.
+
+  For example, you could add a pre-existing Linux bridge (`br0`) with the following command:
+
+        lxc config device add <instance_name> eth0 nic nictype=bridged parent=br0
+- {doc}`Create a managed network </howto/network_create>` and add it as a network device to your instance.
+  With this method, LXD has all required information about the configured network, and you only need to specify the network name when adding the device:
+
+        lxc config device add <instance_name> <device_name> nic network=<network_name>
+
+  If needed, you can add further properties to the command to override the default settings for the network.
+
+## Managed networks
+
+Managed networks in LXD are created and configured with the `lxc network` command.
+
+Depending on the network type, LXD either fully controls the network or just manages an external network interface.
+
+Note that not all {ref}`NIC types <instance_device_type_nic>` are supported as network types.
+LXD can only set up some of the types as managed networks.
+
+### Fully controlled networks
+
+Fully controlled networks provide most functionality, including, for example, the ability to do IP management.
+
 LXD supports the following network types:
 
- - [bridge](#network-bridge): Creates an L2 bridge for connecting instances to (can provide local DHCP and DNS). This is the default.
- - [macvlan](#network-macvlan): Provides preset configuration to use when connecting instances to a parent macvlan interface.
- - [sriov](#network-sriov): Provides preset configuration to use when connecting instances to a parent SR-IOV interface.
- - [ovn](#network-ovn): Creates a logical network using the OVN software defined networking system.
- - [physical](#network-physical): Provides preset configuration to use when connecting OVN networks to a parent interface.
+{ref}`network-bridge`
+: A network bridge creates a connection between different networks or network segments.
```

Comment:
Hrm not sure about this definition, where did it come from?

It feels a bit out of place as although its technically true in a general term, its not what the bridge does in LXD in the usual setup.

This is because in that definition each network "segment" is probably the veth-pair between each instance and the LXD host. But we don't reference them as "networks" we reference them as "nics". 

Because the definition referecnes network segments, it suggests that its primary use is for joining multiple networks together. Which is not what the LXD `bridge` network does in the normal scenario. Its used for connecting instances together into a single network L2 segment.

Things do get a bit more complex because the `bridge` network does accept a `bridge.external_interfaces` setting which will then connect external interfaces to the bridge (which then will connect other network segments to the bridge I suppose). But this is certainly not a common scenario (as in many cases the external network is running its own DNS/DHCP services and running on a different IP subnet and so the managed bridge network's DHCP/DNS services would interfere with it).

Reply:
I suggest we just replace:

> A network bridge creates a connection between different networks or network segments.

With

> A network bridge creates a virtual L2 ethernet switch that instance NICs can connect to, making it possible for them to communicate with each other and the host.